ui/lib/core/addon/components/form-field.hbs

```diff
@@ -6,7 +6,42 @@
 {{! template-lint-configure simple-unless "warn" }}
 <div class="field" data-test-field={{or @attr.name true}}>
   {{#if this.isHdsFormField}}
-    {{! TODO the template logic for the form fields migrated to HDS will appear here }}
+    {{#if @attr.options.possibleValues}}
+      {{#if (eq @attr.options.editType "checkboxList")}}
+        <Hds::Form::Checkbox::Group @name={{@attr.name}} data-test-input={{@attr.name}} as |G|>
+          {{#if this.labelString}}
+            <G.Legend data-test-form-field-label>{{this.labelString}}</G.Legend>
+          {{/if}}
+          {{#if this.showHelpText}}
+            <G.HelperText data-test-help-text>{{@attr.options.helpText}}</G.HelperText>
+          {{/if}}
+          {{#if @attr.options.subText}}
+            <G.HelperText data-test-label-subtext>
+              {{@attr.options.subText}}
+              {{#if @attr.options.docLink}}
+                <DocLink @path={{@attr.options.docLink}}>See our documentation</DocLink>
+                for help.
+              {{/if}}
+            </G.HelperText>
+          {{/if}}
+          {{#each @attr.options.possibleValues as |option|}}
+            <G.CheckboxField
+              class={{if this.validationError "has-error-border"}}
```

Comment:
Does including the `has-error-border` class match with HDS design protocol? I'm not sure we need to carry it over if it doesn't align with other product validations messaging. 🤔

Reply:
Yeah, good point. I just carried over the code, but you're right, we can remove it (it's actually adding a border around the checkbox, which [we don't have in the HDS implementation](https://hds-showcase.vercel.app/components/form/checkbox#content)).

Removed in https://github.com/hashicorp/vault/pull/29958/commits/67beedb555c5e7fd153add64d75ca38d776dc984